### PR TITLE
Updated history calls in futures tut to use new API.

### PR DESCRIPTION
**Pull request by ernestoeperez88**

### CODE REVIEW

File: notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb [hunk 1]
```diff
@@ -63,7 +63,7 @@
     {
      "data": {
       "text/plain": [
-       "Future(1058201601, symbol=u'CLF16', root_symbol=u'CL', asset_name=u'Light Sweet Crude Oil', exchange=u'NYMEX', start_date=Timestamp('2012-01-19 00:00:00+0000', tz='UTC'), end_date=Timestamp('2015-12-21 00:00:00+0000', tz='UTC'), first_traded=None, notice_date=Timestamp('2015-12-23 00:00:00+0000', tz='UTC'), expiration_date=Timestamp('2015-12-21 00:00:00+0000', tz='UTC'), auto_close_date=Timestamp('2015-12-17 00:00:00+0000', tz='UTC'), tick_size=0.01, multiplier=1000.0, exchange_full=u'NYMEX')"
+       "Future(1058201601 [CLF16])"
```
Comment (mmargenot) [inline notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb]: Do we no longer include all the other information as part of the futures object?

File: notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb [hunk 2]
```diff
@@ -403,20 +403,20 @@
...
       "        symbol_reference_date.\n",
       "\n",
-      "    fields : str or list, optional\n",
+      "    fields : str or list\n",
       "        String or list drawn from {'price', 'open_price', 'high', 'low',\n",
-      "        'close_price', 'volume', 'contract'}. Defaults to 'close_price'.\n",
-      "\n",
-      "    frequency : {'daily', 'minute'}, optional\n",
-      "        Resolution of the data to be returned. Defaults to 'daily'.\n",
+      "        'close_price', 'volume', 'contract'}.\n",
```
Comment (mmargenot) [inline notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb]: Default returns OHLCV, right?

Comment (mmargenot) [reply notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb]: (Default being no input to `fields`)

File: notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb (hunk 1 above)
Comment (ernestoeperez88) [reply notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb]: No, we updated the representations of  continuous futures and contracts.

Comment (mmargenot) [reply notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb]: So this is just an update to the representation? Can I still get the `root_symbol` or the `exchange` or `expiration_date`?

File: notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb (hunk 2 above)
Comment (ernestoeperez88) [reply notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb]: Actually, it looks like we removed default parameters. I need to update the wording.

Comment (ernestoeperez88) [reply notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb]: I just realized that's the docstring xD I thought it was part of the notebook/tutorial text. Nothing to update here.

File: notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb (hunk 1 above)
Comment (ernestoeperez88) [reply notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb]: correct.

**Pull request merged**